src/utils.js

```diff
@@ -50,3 +50,7 @@ export const burnItDown = (message?: BurnItDownResponse) => {
     unsubscribe();
   }, timeToBurnItDown);
 };
+
+export const sortByTimestamp = (messages: Array<{timestamp: number}>): Array<Object> => {
```

Comment:
Nitpick: this could be shortened a bit

`export const sortByTimestamp = (messages: Array<{timestamp: number}>): Array<Object> => messages.sort((a,b) => a.timestamp - b.timestamp);`